spring-integration-file/src/test/java/org/springframework/integration/file/locking/FileChannelCacheTests.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
```

Comment:
Well, we stay with a convention to have a first year when the class has been created.